sass-style-guide.md

```diff
@@ -165,90 +172,67 @@ ul
   list-style-type: circle
 ```
 
-### Avoid element selectors
+#### Vermeide Elementselektoren
```

Comment:
* [ ] Wir haben hier einen Namenskonflikt: Sowohl die HTML-Basiselemente als auch die Komponentenelemente (E in BEM) sind Elementselektoren. Wollen wir erstere anders/konkreter benennen?